test/iron-overlay-behavior.html

```diff
@@ -1039,6 +1039,23 @@
 
       });
 
+      suite('animations', function() {
+
+        test('overlay animations correctly triggered', function(done) {
```

Comment:
eheh sarcastic..yes and no! `document.elementFromPoint` is actually a cool way to see what's where without causing a reflow :grin: